firestore/api/SolutionCounter/SolutionCounter.Test/SolutionCounter.Test.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="nunit" Version="3.11.0" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.12.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\SolutionCounter\SolutionCounter.csproj" />
+  </ItemGroup>
+
+</Project>
```

Comment:
Let's move tests into file `C:\git_reps\dotnet-docs-samples\firestore\api\FirestoreTest\FirestoreTest.cs`. Looks like it's more correct place for them